template/src/utils/api.ts

```diff
@@ -1,87 +0,0 @@
-export enum RemoteDataStates {
```

Comment:
The whole logic from `api.ts` file, with `RemoteData`, was replaced by `useQuery` and `useMutation`. This logic is in TanStack Query library.